src/Gui/Splashscreen.cpp

```diff
@@ -235,10 +267,12 @@ void AboutDialogFactory::setDefaultFactory(AboutDialogFactory *f)
  *  The dialog will be modal.
  */
 AboutDialog::AboutDialog(bool showLic, QWidget* parent)
-  : QDialog(parent), ui(new Ui_AboutApplication)
+    : QDialog(parent)
 {
     Q_UNUSED(showLic);
 
+    ui = new Ui_AboutApplication;
+    summaryReport new SummaryReport;

```

Comment:
Invalid syntax. `summaryReport = new SummaryReport;`
Like above this should go to the initialization list